manifest.yml

```diff
@@ -1,13 +1,13 @@
 ---
-name: ANSYS Workbench
+name: ANSYS Workbench (Owens)
```

Comment:
Since we are just replacing the existing ANSYS installation, keep the title the same as before, so remove `(Owens)`